### PR TITLE
Clean up solution file references

### PR DESCRIPTION
**Pull request by blairconrad**
When I opened the sln recently, it complained about missing projects, and I noticed that the workflow yml files were poorly linked.
I removed the references to the bad projects and manually readded the yml files.

### CODE REVIEW

File: FakeItEasy.Analyzers.sln [hunk 1]
```diff
@@ -9,17 +9,17 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		.gitignore = .gitignore
 		.mailmap = .mailmap
 		build.cmd = build.cmd
-		build.sh = build.sh
 		build.ps1 = build.ps1
-		ci.yml
+		build.sh = build.sh
+		.github\workflows\ci.yml = .github\workflows\ci.yml
```
Comment (blairconrad) [inline FakeItEasy.Analyzers.sln:14]: the workflow files will show up right under "Solution Items". If you prefer a folder, I am happy to create and nest. I don't love a ".github/workflows" folder, so would suggest "workflows", but am open to whatever.

File: FakeItEasy.Analyzers.sln [hunk 2]
```diff
@@ -61,26 +61,13 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FakeItEasy.Analyzer.VisualB
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "tools", "tools", "{9D99B251-FB4F-49BF-8405-F6E0DE46EB32}"
 	ProjectSection(SolutionItems) = preProject
-		tools\deploy.cmd = tools\deploy.cmd
-		tools\Directory.Build.props = tools\Directory.Build.props
```
Comment (blairconrad) [inline FakeItEasy.Analyzers.sln:65]: Did not exist.

File: FakeItEasy.Analyzers.sln [hunk 3]
```diff
@@ -61,26 +61,13 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FakeItEasy.Analyzer.VisualB
...
 	EndProjectSection
 EndProject
 Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "FakeItEasy.Analyzer.Shared", "src\FakeItEasy.Analyzer.Shared\FakeItEasy.Analyzer.Shared.shproj", "{E3ED4908-BC8D-4D81-A31B-8CD0B67F41B4}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FakeItEasy.Build", "tools\FakeItEasy.Build\FakeItEasy.Build.csproj", "{81779103-4CA9-4177-B78A-A435FC37DADC}"
-EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FakeItEasy.PrepareRelease", "tools-shared\FakeItEasy.PrepareRelease\FakeItEasy.PrepareRelease.csproj", "{0E2ED1CD-C7ED-4641-9482-C4BBBA58B623}"
-EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FakeItEasy.Deploy", "tools-shared\FakeItEasy.Deploy\FakeItEasy.Deploy.csproj", "{5F4FC5FA-E2FA-456D-813F-6057B7CE7FF0}"
-EndProject
```
Comment (blairconrad) [inline FakeItEasy.Analyzers.sln:77]: None of these exist.

File: FakeItEasy.Analyzers.sln [hunk 4]
```diff
@@ -61,26 +61,13 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FakeItEasy.Analyzer.VisualB
...
 Global
-	GlobalSection(SharedMSBuildProjectFiles) = preSolution
-		tests\FakeItEasy.Analyzer.Tests.Shared\FakeItEasy.Analyzer.Tests.Shared.projitems*{38f3191b-1ce0-4ee0-930f-794a2f2f4cdb}*SharedItemsImports = 4
-		tests\FakeItEasy.Analyzer.Tests.Shared\FakeItEasy.Analyzer.Tests.Shared.projitems*{c37e7b74-dca9-47ff-baa1-ebc2ecb2c72f}*SharedItemsImports = 4
-		src\FakeItEasy.Analyzer.Shared\FakeItEasy.Analyzer.Shared.projitems*{e3ed4908-bc8d-4d81-a31b-8cd0b67f41b4}*SharedItemsImports = 13
-	EndGlobalSection
```
Comment (blairconrad) [inline FakeItEasy.Analyzers.sln:83]: I did not intentionally remove this section. The IDE did it for me. I don't know if it's a problem. Things still build…

**Comment by blairconrad** [comment]
(keeping the solution files up to date is a ginormous pain)

File: FakeItEasy.Analyzers.sln (hunk 1 above)
Comment (thomaslevesque) [reply FakeItEasy.Analyzers.sln:14]: > the workflow files will show up right under "Solution Items".

👍

File: FakeItEasy.Analyzers.sln [hunk 5]
```diff
@@ -61,26 +61,13 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FakeItEasy.Analyzer.VisualB
...
-		tools\deploy.cmd = tools\deploy.cmd
-		tools\Directory.Build.props = tools\Directory.Build.props
 		tools\FakeItEasy.ToolPaths.targets = tools\FakeItEasy.ToolPaths.targets
 		tools\prepare_release.cmd = tools\prepare_release.cmd
 	EndProjectSection
 EndProject
 Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "FakeItEasy.Analyzer.Shared", "src\FakeItEasy.Analyzer.Shared\FakeItEasy.Analyzer.Shared.shproj", "{E3ED4908-BC8D-4D81-A31B-8CD0B67F41B4}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FakeItEasy.Build", "tools\FakeItEasy.Build\FakeItEasy.Build.csproj", "{81779103-4CA9-4177-B78A-A435FC37DADC}"
-EndProject
```
Comment (thomaslevesque) [inline FakeItEasy.Analyzers.sln:73]: Shouldn't we have the FakeItEasy.Analyzers.Build project instead, though?
Not strictly necessary, but we have it in the FIE solution

File: FakeItEasy.Analyzers.sln (hunk 3 above)
Comment (thomaslevesque) [reply FakeItEasy.Analyzers.sln:77]: They should exist...
image
Did you restore the submodule?

Anyway, we probably don't need them in the solution...

File: FakeItEasy.Analyzers.sln (hunk 4 above)
Comment (thomaslevesque) [reply FakeItEasy.Analyzers.sln:83]: It probably doesn't matter. When I just opened the solution and saved, VS also removed it for me.

File: FakeItEasy.Analyzers.sln (hunk 5 above)
Comment (blairconrad) [reply FakeItEasy.Analyzers.sln:73]: Oh, right. Will add.

File: FakeItEasy.Analyzers.sln (hunk 3 above)
Comment (blairconrad) [reply FakeItEasy.Analyzers.sln:77]: Yeah, that was my bad. I didn't restore the submodule until later, and forgot to go back.
I agree, though. No point adding the tools-shared items. Will keep out. And also remove FakeItEasy.ToolPaths.targets, which I missed before!

**Review by thomaslevesque — APPROVED** [review]

**Review by thomaslevesque — APPROVED** [review]

**Comment by blairconrad** [comment]
Thanks, @thomaslevesque.

**Comment by blairconrad** [comment]
Thanks, @thomaslevesque!

**Pull request merged**